02.calendar/cal.rb

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env ruby
+require "optparse"
+require "date"
+
+opt = OptionParser.new
+option = {}
+opt.on( "-m ARG", desc = "set month variable" ){ |m| option[:m] = m }
+opt.on( "-y ARG", desc = "set year variable" ){ |y| option[:y] = y }
+opt.parse(ARGV)
+
+if option[:m] != nil
+  month = option[:m].to_i
+else
+  month = Date.today.month
+end
+
+if option[:y] != nil
+  year = option[:y].to_i
+else
+  year = Date.today.year
+end
+
+first_date = Date.new(year, month, 1)
+last_date = Date.new(year, month, -1)
+range_days = (first_date..last_date)
+space  = "   "
+
+puts "       #{month}月 #{year}        "
```

Comment:
どっちでもいいと言えばどっちでもいいんですが、末尾のスペースは必要なのかなと思いました。

```suggestion
puts "       #{month}月 #{year}"
```